### PR TITLE
177554905 heroku deploy

### PR DESCRIPTION
**Pull request by jkiyasu**
Description: We deployed the heroku for the frontend repo of our project (we have two heroku websites and two repos for the frontend and backend, respectively).

We ran into issues being able to deploy to Heroku and needed to comment out the import "./_map" from the "apps/components/map_component.jsx" file

**@team members: If someone has this file, please let Jenna know**

This is what we did:
1. add a `build` script to `package.json`
  - "build": "expo build:web"
2. Make sure expo-cli is installed as a dev dependency
  - you can just run `npm install -D expo-cli` to get that added
3. Create a Procfile in the root level of the project
  - It should have this one line in it
    - web: npx serve web-build
4. remove yarn.lock
5. push changes to heroku

Our app is live at https://bgb-reactnative.herokuapp.com/

### CODE REVIEW

File: app/components/map/map_component.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import LocationPin from './LocationPin.jsx'
 import { View } from "react-native";
 import HeaderComponent from "../shared/header/header_component";
 import ToolbarComponent from "../shared/toolbar/toolbar_component";
-import { mapStyles } from "./_map";
+//import { mapStyles } from "./_map";
```
Comment (mariela-hdz) [inline app/components/map/map_component.jsx:8]: If anyone has this file please let Jenna know

Comment (lzy-106) [reply app/components/map/map_component.jsx:8]: (PR review assignment:)
What is the specific issue?

File: app/components/map/map_component.jsx [hunk 2]
```diff
@@ -114,7 +114,8 @@ const MapComponent = ({ zoomLevel, navigation}) => {
 
         ))}
 
-      <ToolbarComponent style={mapStyles.navigation} navigation={navigation}></ToolbarComponent>
+      <ToolbarComponent //style={mapStyles.navigation} 
+      navigation={navigation}></ToolbarComponent>
```
Comment (lzy-106) [inline app/components/map/map_component.jsx:118]: (PR review assignment:)
The indentation is not consistent.
Do you have any linter?

**Review by kingdish — COMMENTED** [review]
**What is the goal of this PR?**

To add configuration files for Heroku deployment

**Were you able to view screenshots, story cards, and or test the app?**

Yes. We can visit the Heroku page but there are some problems with the Google Maps integration (“This page can't load Google Maps correctly”).

**Is the PR in a shape that makes it easy to review?**

Mostly. They do have Heroku and detailed descriptions. However, a few changes are not explained. The root cause of removal of `./_map` and related code is not explained.

**What are the status of the tests?**

They don’t have any tests yet. This PR is for deployment, which does not require testing. 

**What is the style of the code like?**

We do not think there’s a code linter. We think the code style is not consistent based on the two changes in `map_component.jsx`.

**Is there anything missing from this PR that should be included?**
They should fix the Google Maps integration bug before merging into the master branch.

**Feature Design: Are there any parts of this feature that are cause for concern?**
This is not applicable since this PR is about deployment.

---
### Overall Assessment

**What does this PR do well?**
It gives the Heroku link for us to review. The description is pretty nice and detailed.

**What is something that could be improved with this PR?**
A few descriptions are not detailed enough for a few changes. There’s a bug that needs to be fixed. The code style is a bit inconsistent, such as the indentation.

**Pull request merged**